docs/privacy/general-privacy.md

```diff
@@ -1,60 +1,139 @@
+any other means of identification.
+Be aware that centralized exchanges can be hacked, potentially exposing the
+collected information to the hacker.
+It’s also possible that at some point an oppressive government will seize the
+collected information.
+Cryptocurrency ATMs can be equipped with cameras, fingerprint scanners, or there
+could be cameras nearby the ATM which could reveal your identity.
 
-## Centralized Mixers
+---
 
-The aim of a cryptocurrency mixer is to break the transaction graph. An old version of cryptocurrency mixers is the centralized mixers (also called scrambler, thumbler, shuffler). 
-Either we use a centralized mixer or the later mentioned solutions, [amount correlation](https://en.bitcoin.it/wiki/Privacy#Amount) is a problem.
+## Avoid Leaving Identifiers in Public Places
 
-## Non-custodial Coin Mixing
+Many times this is something physical, for example leaving behind a printed QR code somewhere, writing your nickname or real name on a piece of paper. Your name can be inprinted on the next page of the sticky notes stack, for example.
+Be careful not to leave your phone or computer unlocked in a public place with
+important information displayed on screen.
 
-It is possible to mix coins without a centralized exchange. One method is [TumbleBit](https://github.com/decred/tumblebit), which allows parties to make payments through an untrusted Tumbler.
-[Decred's build in coin mixing](https://blog.decred.org/2019/08/28/Iterating-Privacy/) feature is based on the [CoinShuffle++](https://github.com/decred/cspp) protocol.
+---
 
 ## Using the Lightning Network for Privacy
 
-Privacy attacks like [common-input-ownership heuristic](https://en.bitcoin.it/wiki/Privacy#Common-input-ownership_heuristic), [address reuse](https://en.bitcoin.it/wiki/Privacy#Address_reuse) and change address detection fundamentally don’t work on any off-chain transactions technology. The Lightning Network is an off-chain transaction technology. One possible downside is that the operator of a lightning node could keep logs of the pseudonymous identifiers, the balances, and the changes of the balances of the participants. This node could later sell this information to governments or corporations. A research paper called [BOLT](https://eprint.iacr.org/2016/701) proposes solutions for this problem.
+Privacy attacks like [common-input-ownership
+heuristic](https://en.bitcoin.it/wiki/Privacy#Common-input-ownership_heuristic),
+[address reuse](https://en.bitcoin.it/wiki/Privacy#Address_reuse) and change
+address detection fundamentally don’t work on any off-chain transaction
+technology such as The Lightning Network.
+One possible downside is that the operator of a lightning node could keep logs
+of the pseudonymous identifiers and the balances of the network participants.
+The node operator could later sell this information to governments or corporations.
+A research paper called [Bolt](https://eprint.iacr.org/2016/701) proposes
```

Comment:
While I'm sure this Bolt research paper is useful, I'm wondering if this reference will age well. If not implemented or otherwise obsoleted, could see this ending up as kruft that never gets removed.